solr/core/src/test/org/apache/solr/cloud/TestBaseStatsCacheCloud.java

```diff
@@ -111,23 +111,23 @@ public void testBasicStats() throws Exception {
                 "q", "foo_t:\"bar baz\"", "fl", "*,score", "rows", "" + NUM_DOCS, "debug", "true"));
     QueryResponse controlRsp =
         control.query(
-            "collection1",
+                DEFAULT_TEST_COLLECTION_NAME,
```

Comment:
probably need `./gradlew tidy` due to indentation btw

Reply:
Thanks!   Doing it now.